frontend/src/container/NotificationWall.jsx

```diff
@@ -425,7 +424,7 @@ export const NotificationWall = props => {
     }
 
     if (entityType === TLM_ENTITY.MENTION && eventType === TLM_EVENT.CREATED) {
-      const groupMention = GROUP_MENTION_TRANSLATION_LIST.includes(notification.mention.recipient)
+      const groupMention = notification.mention.type === 2
```

Comment:
2 feels like a magic number

Reply:
Indeed. 2 represent that the mention type of a role mention. This is defined in the backend. Do I create a ``mention.type`` definition in ``mentionOrLink.js``? Do I update the code to specify what is the ``2`` in a comment?

Reply:
I think a definition would be better, but a comment is good enough

Reply:
Done in [fb7c4c5](https://github.com/tracim/tracim/pull/6139/commits/fb7c4c58c03689f463e6df01d34e4cf8152d55ab)